src/main/java/net/hypixel/nerdbot/command/ReminderCommands.java

```diff
@@ -110,6 +95,48 @@ public void createReminder(GuildSlashEvent event, @AppOption(description = "Use
         }
     }
 
+    @JDASlashCommand(name = "remind", subcommand = "edit", description = "Edit an existing reminder")
+    public void editReminder(GuildSlashEvent event, @AppOption(description = "Can be obtained from /remind list") UUID uuid, @AppOption(description = "The new content of your reminder") String description, @Optional @AppOption(description = TIME_DESCRIPTION) String time) {
+        Reminder reminder = NerdBotApp.getBot().getDatabase().findDocument(NerdBotApp.getBot().getDatabase().getCollection("reminders", Reminder.class), Filters.eq("uuid", uuid)).first();
+
+        if (reminder == null) {
+            event.reply("Could not find a reminder with that ID!").setEphemeral(true).queue();
+            return;
+        }
+
+        if (!reminder.getUserId().equals(event.getUser().getId())) {
+            event.reply("You can only edit your own reminders!").setEphemeral(true).queue();
+            return;
+        }
+
+        Date date;
+        try {
+            date = parseTime(time);
+        } catch (DateTimeParseException exception) {
+            event.reply(TIME_PARSE_ERROR).setEphemeral(true).queue();
+            return;
+        }
+
+        if (date != null) {
+            reminder.setTime(date);
+            if (reminder.getTimer() != null) {
+                reminder.getTimer().cancel();
+            }
+        }
+
```

Comment:
If the date happens to be null, the user will see that the time was updated when it didn't.